contracts/src/pre-deploy/L2BaseFeeVault.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { FeeVault } from "src/libraries/FeeVault.sol";
+
+/// @custom:proxied
+/// @custom:predeploy 0x2A00000000000000000000000000000000000013
+/// @title L2BaseFeeVault
+/// @notice The L2BaseFeeVault accumulates the base fee that is paid by transactions.
+contract L2BaseFeeVault is FeeVault {}
```

Comment:
I created an implementation of `FeeVault` for `L2BaseFeeVault`, because the goroutine that generates genesis assumes the name of the pre-deploy matches the name of the contract implementation. We'll also want to deploy vaults for the `L1Fee` and `SequencerFee`.

I can change that script if this isn't the right way to do this.